awscli/help.py

```diff
@@ -250,6 +259,10 @@ def __init__(self, session, command_table, arg_table,
         self.description = description
         self.synopsis = synopsis
         self.help_usage = usage
+        self._topic_table = None
+        self._topic_tag_db = None
+        self._related_items = [
+            'AWS CLI Topic Guide (`aws help topics <../topic/index.html>`_)']
```

Comment:
I think we're mixing layers here.  Prior to this, the help commands knew nothing about how the documentation was actually going to be formatted, nor did they contain any RST markup.  This seems like a concern of bcdoc and the clidoc event handlers.